apps/meteor/server/routes/avatar/utils.ts

```diff
@@ -119,22 +120,21 @@ export async function userCanAccessAvatar({ headers = {}, query = {} }: IIncomin
 	return isAuthenticated;
 }
 
-const getFirstLetter = (name: string) =>
-	name
-		.replace(/[^A-Za-z0-9]/g, '')
-		.substr(0, 1)
-		.toUpperCase();
+const getFirstLetter = (name: string) => {
+	const sanitizedName = DOMPurify.sanitize(name);
+	return sanitizedName.substring(0, 1).toUpperCase();
+};
 
-export const renderSVGLetters = (username: string, viewSize = 200) => {
+export const renderSVGLetters = (roomOrUsername: string, viewSize = 200) => {
```

Comment:
can't we just call this 'name'? if a room is passed down it would be the room name right? this way it is more agnostic of where it is being used while still having semantic meaning when we do `initials = name`

Reply:
I'll change it to “name” as suggested.